apps/public-reference/pages/applications/contact/alternate-contact-type.tsx

```diff
@@ -31,13 +36,10 @@ export default () => {
     application.alternateContact.type = data.type
     application.alternateContact.otherType = data.otherType
 
-    conductor.completeStep(1)
+    if (data.type === "noContact") conductor.completeSection(1)
```

Comment:
I'm not 100% sure about our style guide on this matter, but most JS projects I've worked on don't like single line/block-less `if` statements - since whitespace isn't significant in JS, it's too potentially error prone. I know you're coming (mostly) from Ruby, so I understand the temptation for nice concise code ;-) tho...

Reply:
I don't think we have a formal style guide other than to rely on the Prettier config. Did a quick regex search and there's one-line if statements a number of other places as well. I understand the rationale for wanting to standardize around multi-line blocks, but I would say I'm in favor of both options. (And yes, in Ruby the conditional is added to the end of the line where it belongs. 😉)